nc2zarr/processor.py

```diff
@@ -23,11 +23,13 @@
 
 import xarray as xr
 
+_UNDEFINED = object()
+
 
 class DatasetProcessor:
     def __init__(self,
                  process_rename: Dict[str, str] = None,
-                 process_rechunk: Dict[str, Dict[str, int]] = None,
+                 process_rechunk: Dict[str, Any] = None,
```

Comment:
>  Is this intentional?

Yes. This is a private implementation class that is only used from within the Converter class. If this was API, I'd may have used the complex type.

> Also, is there a reason to declare types as Dict rather than the more general Mapping (both here and elsewhere)?

Yes, because it is an implementation class that _will_ be called with a dict. If this was API, I'd used `Mapping`.